### DISCUSSION TITLE
create SSH rule

### DISCUSSION THREAD

**Discussion by seifertcl** [Security-Onion-Solutions/securityonion#7374] [General] [answered]
Can you create an ssh rule that detects ssh traffic that is not version2.

**Comment by dougburks** [comment, marked as answer]
Zeek already logs ssh connections including the version and Hunt includes an SSH query for that:
![Screen Shot 2022-03-02 at 3 04 37 PM](https://user-images.githubusercontent.com/1659467/156440967-a3a47017-517b-46aa-a2e1-f40abb660737.png)

Another option might be to write a Suricata rule:
https://suricata.readthedocs.io/en/suricata-6.0.0/rules/ssh-keywords.html
